.github/workflows/reviewdog.yml

```diff
@@ -0,0 +1,31 @@
+name: reviewdog
+
+on:
+  pull_request:
+    branches: ["main"]
+
+jobs:
+  check_todo_in_this_commit:
+    name: Check TODO_IN_THIS_COMMIT
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: okdas/action-fail-on-found@v1
```

Comment:
@okdas where is the code for this?

Reply:
Hey @Olshansk and @bryanchriswhite - the GitHub action itself resides in a separate repo - https://github.com/okdas/action-fail-on-found. Not much code to show - it is just a simple [one-liner](https://github.com/okdas/action-fail-on-found/blob/master/entrypoint.sh#L12) though that uses `the_silver_searcherer` to find mentions in the code and pipes the output into the [reviewdog](https://github.com/reviewdog/reviewdog). 
Let me know if you want to transfer the repo of this action into `pokt-network` organization, so we can all maintain it. Originally, I didn't think this was necessary.

Reply:
Yea, let's move it over just so it's all in one place.